packages/kbn-openapi-generator/src/template_service/templates/zod_schema_item.handlebars

```diff
@@ -68,7 +68,6 @@ z.unknown()
 
 {{~#*inline "type_boolean"~}}
   z.boolean()
-  {{~#if nullable}}.nullable(){{/if~}}
```

Comment:
@xcrzx I've noticed this line is redundant here.  With it in place `index_mapping_outdated ` field in `read_index.gen.ts` gets double `.nullable().nullable()`.

Reply:
Okay, thanks 👍